packages/component-library/src/PDF/PDF.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { css } from 'emotion';
+
+const pdfClass = css`
+  width: 100%;
+  height:calc(100vh - 70px);
```

Comment:
Using `vh` here seems weird. What's the goal with this?

Reply:
Was trying to make it so that the PDF embed fills most of, but not all of the screen on desktop and mobile. There’s probably a better way to do this. Will revisit.